UPGRADE-4.4.md

```diff
@@ -104,6 +104,9 @@ HttpFoundation
 
  * `ApacheRequest` is deprecated, use `Request` class instead.
  * Passing a third argument to `HeaderBag::get()` is deprecated since Symfony 4.4, use method `all()` instead
+ * [BC BREAK] `PdoSessionHandler` with MySQL changed the type of the lifetime column,
+   make sure to run `ALTER TABLE sessions MODIFY sess_lifetime INTEGER UNSIGNED NOT NULL` to
+   update your database.
```

Comment:
Can you please add this to `UPGRADE-5.0.md` and to the CHANGELOG of the component also?

Reply:
done